inc/zoo/map/RobinHood.h

```diff
@@ -102,71 +112,6 @@ struct RH_Backend {
         return startingPSLmadePotentialPSLs;
     }
 
-    template<int Misalignment, typename KeyComparer>
```

Comment:
If unaligned find is easy (which it is), we should keep both styles of find around, present both, and present benchmarks on both (if time allows)

Reply:
This code drifted so much from the head that it needed removal to prevent confusion